tests/t/real.json

```diff
@@ -0,0 +1 @@
+3.1415926500000002
```

Comment:
wait, a double should be able to go out to 3.141592653589793 -- are we losing precision here?